infrastructure/terraform.test.tfvars

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2023, Xgrid Inc, https://xgrid.co
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+namespace      = "testing"
+region         = "eu-west-1"
+account_id     = "123456789012"
+vpc_cidr_block = "15.0.0.0/16"
+public_subnet_cidr_block = {
+  "eu-west-1a" = "15.0.1.0/24"
+  "eu-west-1c" = "15.0.2.0/24"
+}
+domain_name                = "example.co"
```

Comment:
For testing environment, we can bypass cognito and SSL certificates. What do you say?

Reply:
Added in fa21638e1999d560021571836a2324feed3ec634